src/Aspire.Hosting.Keycloak/KeycloakResourceBuilderExtensions.cs

```diff
@@ -163,14 +164,38 @@ public static IResourceBuilder<KeycloakResource> WithRealmImport(
 
         if (Directory.Exists(importFullPath))
         {
-            return builder.WithBindMount(importFullPath, RealmImportDirectory, isReadOnly);
+            return builder.WithContainerFiles(
+                KeycloakDataDirectory,
+                [
+                    new ContainerDirectory
+                    {
+                        Name = "import",
+                        Entries = Directory.GetFiles(importFullPath)
```

Comment:
It's a bit unfortunate this code needs to be written. It feels like we should just be able to point to a directory, and the whole thing is used, like a bind mount.

Reply:
I definitely need to add some helpers to simplify this particular pattern, but one of the reasons for using the cp command over bind mounts is to allow explicit control over file permissions (such as ensuring scripts are marked executable). This requires us to be able to override permissions on per-file basis, so we explicitly chose a more verbose API. We don’t want a 1:1 replacement for WithBindMount as this is intended to be used for specific advanced scenarios. 

One of the limitations of the `cp` command is that the root target directory in the container has to exist, but child directories are treated as create or update. The `import` directory doesn’t exist by default in KeyCloak containers so can’t be part of the root destination path.

Reply:
Implemented a helper method to simplify this particular pattern.